atomfeeds.php

```diff
@@ -49,6 +49,23 @@ function atomfeeds_civicrm_enable() {
   _atomfeeds_civix_civicrm_enable();
 }
 
+/**
+ * Implements hook_civicrm_container().
+ */
+function atomfeeds_civicrm_container(\Symfony\Component\DependencyInjection\ContainerBuilder $container) {
+  $container->setDefinition("cache.dashboard", new Symfony\Component\DependencyInjection\Definition(
```

Comment:
I wonder whether we're changing a pre-existing cache.dashboard cache here. If so, this might not be what we want; this might interfere with other extensions, or maybe core CiviCRM. (As said, it's been a while, so I'm not sure.)
Maybe we should create a cache with a less generic name, e.g. `cache.atomfeeds`, to reduce the risk of changing the configuration of some other existing caching service.